src/components/v1/controller.v1.ts

```diff
@@ -46,7 +46,25 @@ const downloadSVG = async (req: Express.Request, res: Express.Response, next: Ex
   }
 }
 
+const iconsApi = async (req: Express.Request, res: Express.Response, next: Express.NextFunction) => {
```

Comment:
can we change the name of this controller? to something like `fontsApi`, I know the route has iconsapi but it doesnt make any sense now as all are icons api :p
we can keep the route same now